src/ray/core_worker/transport/direct_task_transport.cc

```diff
@@ -423,15 +422,54 @@ CoreWorkerDirectTaskSubmitter::GetOrConnectLeaseClient(
   return lease_client;
 }
 
+void CoreWorkerDirectTaskSubmitter::ReportWorkerBacklog() {
+  absl::MutexLock lock(&mu_);
+  std::unordered_map<SchedulingClass, std::pair<TaskSpecification, int64_t>> backlogs;
```

Comment:
```suggestion
  absl::flat_hash_map<SchedulingClass, std::pair<TaskSpecification, int64_t>> backlogs;
```

prefer absl maps when possible for performance